hudi-client/src/main/java/org/apache/hudi/config/HoodieIndexConfig.java

```diff
@@ -166,6 +168,11 @@ public Builder bloomIndexBucketizedChecking(boolean bucketizedChecking) {
       return this;
     }
 
+    public Builder bloomIndexEnableCompression(boolean enableCompression) {
+      props.setProperty(BLOOM_INDEX_ENABLE_COMPRESSION, String.valueOf(enableCompression));
```

Comment:
just call Boolean.toString() instead of String.valueOf()

Reply:
Done.